PayPalUI/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
These get auto generated we can delete since they're handled by top-level `.gitignore`.